new-pipeline/src/dashboards.js

```diff
@@ -1077,7 +1077,14 @@ function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = buildDictionaryURL(metric, channel, description);
   var descr = metric;
   if (description && (description.length > 0)) {
-    descr = description;
+	  
+	  if (description.length > 200) {
+		  descr = description.substring(0, 200);
+		  descr += ' ...'
```

Comment:
This has roughly the desired effect, but the better approach is to solve this using CSS.
We should probably use styling with `text-overflow: ellipsis;` etc. to achieve this effect.
https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
Um, ok :+1: